src/recyclerlistview/RecyclerListView.js

```diff
@@ -339,8 +347,8 @@ class RecyclerListView extends Component {
         }
         if (!this._nextFrameRenderQueued) {
             this._nextFrameRenderQueued = true;
-            if (requestAnimationFrame) {
-                requestAnimationFrame(() => {
+            if (relayoutRequestThrottler) {
+                relayoutRequestThrottler(() => {
```

Comment:
This makes changes to DOM rite?? If yes then y not rAF instead of rIC??? https://developers.google.com/web/updates/2015/08/using-requestidlecallback#using_requestidlecallback_to_make_dom_changes

Reply:
So this particular thing needs to get triggered when there is a mismatch between estimated heights provided and the actual rendered ones. Basically a relayout. By throttling I want to wait a few frames to know what layouts are thrashed and run layout when thread has some free time. rAF gives me no time and clogs the main thread on web in case of non deterministic rendering. rIC is working very well on web and iOS. It's strange however that it actually slows down stuff on Android. Any other ideas?